test/suites/test-10-ecma.js

```diff
@@ -0,0 +1,32 @@
+define(['pumascript', 'esprima'], function(puma, esprima) {
+//section 10.1	
+  test("Use Strict global code", function () {
+    var result = puma.evalPuma("\'use strict\';var a = 1;
+var b = 3;
+console.log(a+b);");
+    ok(result.success && '4' === result.value, "Pass!");
+  });
```

Comment:
Add new line (space) between lines 8 - 9, to improve readability and organization